### DISCUSSION TITLE
Mixed Effect Model

### DISCUSSION THREAD

**Discussion by msoheib** [flatironinstitute/nemos#261] [Q&A]
I have 2-photon calcium imaging data from many mice from different intervention groups. To control for effects between different subject etc our lab usually chooses Mixed Effect models. Is there support for using the Mixed Effect model in Nemos?

If not, does anyone have a recommendation to do this?

**Comment by BalzaniEdoardo** [comment]
Hi, thanks for reaching out!

Currently, NeMoS do not implement mixed effect models, but it may be an interesting and useful feature to have. Right now, I can point you to [statsmodels](https://www.statsmodels.org/stable/mixed_glm.html) for a python implementation of GLM with mixed effects.

It would be nice if you can share a bit more on the type of model you would like to set up, and can you point to an example in literature using mixed models. 

What is the framework that your lab usually works with? 

Best,
Edoardo
